src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -95,7 +105,10 @@
        (merge
         {:db (cond-> (assoc-in db' [:wallet :send-transaction] {})
 
-               (not= method constants/web3-personal-sign)
+               (not (#{constants/web3-sign-typed-data
+                       constants/web3-sign-typed-data-v3
+                       constants/web3-personal-sign}
```

Comment:
Looks duplicated? Can we reuse an extracted fn?

Reply:
what do you mean by duplication?

Reply:
Several time the same checks, could be a separate fn?